client/card-readers/settings/index.tsx

```diff
@@ -20,9 +20,9 @@ import AddressDetailsSection from './sections/address-details';
 import BrandingDetailsSection from './sections/branding-details';
 import { getAdminUrl } from 'wcpay/utils';
 
-const isBrandingEnabled = false;
+const isBrandingEnabled = true;
```

Comment:
Is this intentional?

Reply:
Sorry, it is a mistake. I've changed it for testing purposes and forgot to set it back to `false`. Sorted.